releasenotes/notes/remove_8070-09b64e3a2aca1c2d.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The arguments ``x``, ``z`` and ``label`` to the initializer of
+    :class:`.Pauli` were removed, as deprecated in Qiskit Terra 0.17 (released on 2021-04-02).
+  - |
+    Importing from ``qiskit.quantum_info.operators.pauli`` was deprecated in Qiskit Terra 0.21
+    (released on 2022-06-30) is now removed.
```

Comment:
For both of these it would be good to document the alternative people should be using instead. It is really useful for users when reading the release notes (typically after something breaks) to see how they should update their code based on this change.

Reply:
yeap! added in 9de7412a01e77d18bf9ec4bce4cd85192d34c78a